src/components/Filter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { STATUSES, AllProductRequestProps, ProductRequest } from 'types';
+import '../index.scss';
+
+interface IProps {
+  productRequests: ProductRequest[];
+  showData: ProductRequest[];
+  setShowData: any;
+}
+
+export const Filter = (props: IProps) => {
+  const { productRequests } = props;
+  const { showData } = props;
+  const { setShowData } = props;
+
+  const showAll = () => {
+    setShowData(productRequests);
+  };
+
+  const filter = (productCategory: string) => {
```

Comment:
Suggestion to add a return type here

Reply:
would the return type be void since it's not actually returning anything? It's just updating state